recipe/meta.yaml

```diff
@@ -10,17 +10,17 @@ source:
   sha256: 333472e2562343dfdd27407ec9b5114a07c81d0416794e4ac6b703dd925c6a1a
 
 build:
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: True  #[py<38]
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - hatchling
     - pip
   run:
-    - python >={{ python_min }}
+    - python
```

Comment:
Can you add optional dependencies that have pinnings to `run_constrained` https://github.com/narwhals-dev/narwhals/blob/v1.31.0/pyproject.toml#L31-L40?

Reply:
Got it! Added :)